worker-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/src/main/config/cfg~caf~worker~__workerName__Configuration.js

```diff
@@ -18,6 +18,8 @@
     workerVersion: "${project.version}",
     outputQueue: getenv("CAF_WORKER_OUTPUT_QUEUE")
             || (getenv("CAF_WORKER_BASE_QUEUE_NAME") || getenv("CAF_WORKER_NAME") || "worker") + "-out",
+    rejectQueue: getenv("CAF_WORKER_REJECT_QUEUE")
+            || (getenv("CAF_WORKER_BASE_QUEUE_NAME") || getenv("CAF_WORKER_NAME") || "worker") + "-reject",
```

Comment:
It's fine to change the archetype but don't forget that it only effects future workers.  For existing workers which haven't had this line added `rejectQueue` will be `null`.  The code will have to deal with this.